### PR TITLE
Fixed the broken Dockerfile

### PR DESCRIPTION
**Pull request by philippkahr**
apt-get is not available on alpine.
curl was also missing.
The url to the downloads was wrong as the version is prefixed with a `v` as in `/v1.19.0` instead of `/1.19.0`

### CODE REVIEW

**Comment by philippkahr** [comment]
Only thing, would it be possible to automatically detect the latest version of the agent for the docs? Currently this is hardcoded in line 1:

```
ARG AGENT_VERSION=1.19.0
```

I think the docs should auto update the 1.19.0, every time there is a new release.

**Review by gregkalapos — APPROVED** [review]
LGTM.

@bmorelli25 can you answer [this part](https://github.com/elastic/apm-agent-dotnet/pull/1965#issuecomment-1369666048)? - unfortunately I don't know if we can do that.

File: docs/setup-auto-instrumentation.asciidoc [hunk 1]
```diff
@@ -120,19 +120,18 @@ can be used as part of a https://docs.docker.com/develop/develop-images/multista
...
 ARG AGENT_VERSION
 WORKDIR /source
 
 # install unzip
-RUN apt-get -y update && apt-get -yq install unzip
+RUN apk update && apk add zip curl
 
 # pull down the zip file based on ${AGENT_VERSION} ARG and unzip
-RUN curl -L -o elastic_apm_profiler_${AGENT_VERSION}.zip https://github.com/elastic/apm-agent-dotnet/releases/download/${AGENT_VERSION}/elastic_apm_profiler_${AGENT_VERSION}.zip && \ 
+RUN curl -L -o elastic_apm_profiler_${AGENT_VERSION}.zip https://github.com/elastic/apm-agent-dotnet/releases/download/v${AGENT_VERSION}/elastic_apm_profiler_${AGENT_VERSION}.zip && \
```
Comment (gregkalapos) [inline docs/setup-auto-instrumentation.asciidoc:133]: Ahh, nice!

At some point we added `v` before the version numbers in the .NET agent, in order to be aligned with other APM agents. So the url without `v` worked for older versions, but it's now needed here for newer versions.

**Comment by bmorelli25** [comment]
Apologies, I've been on PTO.

> Only thing, would it be possible to automatically detect the latest version of the agent for the docs?

Hmmm. Unfortunately, there's no way to do this that I'm aware of.

**Pull request merged**